.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: scala
 scala:
-  - 2.11.7
+  - 2.11.8
+  - 2.10.6
 jdk:
   - oraclejdk8
 
 script:
-  - sbt +clean coverage +test coverageReport
+  - sbt clean coverage test coverageReport
```

Comment:
The sample project for cross-compiled projects in scoverage doesn't use `+`, they rely on the scala parameter in the `.travis.yml` file